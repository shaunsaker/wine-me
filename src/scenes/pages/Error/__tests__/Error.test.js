import React from 'react';
import renderer from 'react-test-renderer';

import { Error } from '..';

describe('Error', () => {
  describe('renders', () => {
    it('renders with minimum required props', () => {
      const component = renderer.create(<Error />);

      expect(component).toMatchSnapshot();
    });
  });
});
