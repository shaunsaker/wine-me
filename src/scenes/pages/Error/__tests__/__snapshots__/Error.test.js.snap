// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders renders with minimum required props 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "paddingBottom": 0,
        "paddingTop": 22,
      },
      Object {
        "justifyContent": "center",
      },
      Object {
        "alignItems": "center",
      },
      Object {
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "zIndex": 100,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "height": 22,
          },
          Object {
            "backgroundColor": "#48466D",
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "alignSelf": "stretch",
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#212121",
            "fontSize": 24,
            "marginBottom": 8,
          },
          undefined,
        ]
      }
    >
      Error
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#757575",
            "fontSize": 17,
          },
          undefined,
        ]
      }
    >
      Something went wrong.
    </Text>
  </View>
</View>
`;
